website/core/Footer.js

```diff
@@ -132,37 +131,31 @@ class Footer extends React.Component {
           <div>
             <h5>More</h5>
             <a href="https://www.parity.io/blog/">Blog</a>
-            <a href="https://github.com/paritytech/substrate">Substrate GitHub</a>
-            <a href="https://github.com/substrate-developer-hub/">Developer Hub GitHub</a>
-            <a
-              className="github-button"
-              href={this.props.config.repoUrl}
-              data-icon="octicon-star"
-              data-count-href="/facebook/docusaurus/stargazers"
-              data-show-count="true"
-              data-count-aria-label="# stargazers on GitHub"
-              aria-label="Star this project on GitHub"
-            >
-              Star
+            <a href="https://github.com/paritytech/substrate">
+              Substrate GitHub
+            </a>
+            <a href="https://github.com/substrate-developer-hub/">
+              Developer Hub GitHub
+            </a>
+            <a href="https://www.parity.io/privacy/">Privacy Policy</a>
+            <a href="#" id="cookie-settings">
+              Cookie Settings
             </a>
           </div>
         </section>
 
-        <section className="row">
-          <div className="copyright">{this.props.config.copyright}</div>
-          <a className="item" href="https://www.parity.io/privacy/">Privacy Policy</a>
-          <a className="item" href="#" id="cookie-settings">Cookie Settings</a>
-          <script
-            dangerouslySetInnerHTML={{
-              __html: `
+        <section className="copyright">{this.props.config.copyright}</section>
+
+        <script
```

Comment:
Will it be better if put the script near the element?

Reply:
it doesn't matter right?

I could put a comment on the script saying they are related

Reply:
I moved the script right into the `a` tag and also put a note :)